WordPress/Sniffs/WhiteSpace/ControlStructureSpacingSniff.php

```diff
@@ -506,6 +507,16 @@ public function process_token( $stackPtr ) {
 			return;
 		}
 
+		if ( T_COMMENT === $this->tokens[ $trailingContent ]['code'] ) {
+			if ( $this->tokens[ $trailingContent ]['line'] === $this->tokens[ $scopeCloser ]['line'] ) {
+				if ( '//end' === substr( $this->tokens[ $trailingContent ]['content'], 0, 5 ) ) {
```

Comment:
Just ran some tests and interestingly enough, that causes yet another "case of the disappearing comment" similar to #976 

Looking at the history, this sniff was original cloned from the Squiz standard and the fixers were build in afterwards in 2014. Since the original cloning, the upstream sniff also gained fixers.
I know I've synced a number of relevant changes from upstream in before, but don't think that part has changed that much. Will have a look and see what we can learn from the upstream version.